src/Server.cpp

```diff
@@ -309,7 +304,9 @@ void Server::buildResponse(Connection &conn, size_t &i, HTTPRequest &request, HT
 
 void Server::writeToClient(Connection &conn, size_t &i, HTTPResponse &response)
 {
-	std::cout << "\033[1;36m" << "Entering writeToClient" << "\033[0m" << std::endl;
```

Comment:
Using your color macros will save us from this edge case autoformatter discrepancy